xfel/ui/components/ebeam_plotter.py

```diff
@@ -4,7 +4,8 @@
 from xfel.cxi.cspad_ana import cspad_tbx
 from matplotlib import pyplot as plt
 import numpy as np
-from simtbx.nanoBragg.utils import ENERGY_CONV
+from scipy import constants
+ENERGY_CONV = 10000000000.0 * constants.c * constants.h / constants.electron_volt
```

Comment:
Could this be `factor_kev_angstrom` or `factor_ev_angstrom`?  E.g. `from cctbx import factor_kev_angstrom`

Reply:
```
In [10]: utils.ENERGY_CONV, factor_kev_angstrom*1000
Out[10]: (12398.419843320027, 12398.419843320025)
```

Reply:
Id say either way